frontend/src/concepts/pipelines/topology/core/customNodes/StandardTaskNode.tsx

```diff
@@ -0,0 +1,30 @@
+import * as React from 'react';
+import {
+  TaskNode,
+  DEFAULT_WHEN_OFFSET,
+  Node,
+  WhenDecorator,
+  NodeModel,
+  WithSelectionProps,
+} from '@patternfly/react-topology';
+import { StandardTaskNodeData } from '~/concepts/pipelines/topology/core/types';
+
+type DemoTaskNodeProps = WithSelectionProps & {
+  element: Node<NodeModel, StandardTaskNodeData>;
+};
+
+const StandardTaskNode: React.FC<DemoTaskNodeProps> = ({ element, onSelect, selected }) => {
+  const data = element.getData();
+
+  const whenDecorator = data?.whenStatus ? (
+    <WhenDecorator element={element} status={data.whenStatus} leftOffset={DEFAULT_WHEN_OFFSET} />
```

Comment:
I should look to see if we can get this going before we feature complete -- this could add value to our topology.